packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -182,27 +164,6 @@ export class SummarizerNodeWithGC extends SummarizerNode implements IRootSummari
 		}
 	}
 
-	public async summarize(
```

Comment:
This was overridden to just validate that GC runs. This validation happens in complete summary step anyway. Removing this from here to make the code simpler.

Reply:
What about the ternary return expression that would call `summarizeFn` directly if `!trackState`?  `super.summarize` doesn't check `trackState` at all, so this looks like a behavior change.

Reply:
Well, assuming `trackState === !fullTree`, then the only change (if `trackSate` is false) is that `this.wipLocalPaths` won't be updated.  Can you help me understand what the implications are there and why it's the right change?

Reply:
You are right. I missed updating super.summarize to check for trackState.

Reply:
But if `trackState` were false, it used to be that this would only call `summarizeFn`.  Now it will call `super.summarize`, which will also update `this.wipLocalPaths`. Is that what you want?

Reply:
damn! It doesn't really matter because `trackState` cannot be set to false expect for in tests if it  calls `summarize` directly on container runtime. Anyway, I preserved the original behavior but moved the logic to `summarizerNode` because `summarizerNodeWithGC` handling trackState was incorrect.